modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/planner/AbstractPlannerTest.java

```diff
@@ -653,10 +653,10 @@ protected static IgniteSchema createSchema(TestTable... tbls) {
         private final RewindabilityTrait rewindable;
 
         /** */
-        private final double rowCnt;
+        private final TableDescriptor desc;
 
         /** */
-        private final TableDescriptor desc;
+        private ObjectStatisticsImpl stat;
```

Comment:
can we have this `final`?

Reply:
no, we can change statistics or remove it at all for the given table.